src/Application/Job/CatalogDumpJob.php

```diff
@@ -0,0 +1,132 @@
+
+        foreach ($graph->resources() as $resource) {
+            # Step 2 - get all datasets which are part of the data catalog
+            $datasets = $resource->allResources("schema:dataset");
+            foreach ($datasets as $dataset) {
+                $dataset_uri = $dataset->getUri();
+                $graph->parse($dataset_uri, 'jsonld');
+            }
+
+            # Step 3 - get all distribution which are part of datasets
+            $distributions = $resource->allResources("schema:distribution");
+
+            foreach ($distributions as $distribution) {
+                $distribution_uri = $distribution->getUri();
+                $graph->parse($distribution_uri, 'jsonld');
+            }
+
+            # Step 4 - get all publishers and creators (of data catalog and datasets)
+            $publishers = $resource->allResources("schema:publisher");
+
+            foreach ($publishers as $publisher) {
+                $publisher_uri = $publisher->getUri();
+                $graph->parse($publisher_uri, 'jsonld');
+            }
+
+            $creators = $resource->allResources("schema:creator");
+
+            foreach ($creators as $creator) {
+                $creator_uri = $creator->getUri();
+                $graph->parse($creator_uri, 'jsonld');
+            }
+        }
+
+        # Step 5 - remove Omeka classes and properties (o:)
+        $this->removeOmekaTags($graph);
+
+        # Step 6 - output the graph in several serializations
+        $this->dumpSerialisedFiles($graph);
+    }
+
+    // Maybe this step isn't needed, Bob said on 11th April.
+    protected function removeOmekaTags(Graph $graph): void
+    {
+        foreach ($graph->resources() as $resource) {
+            foreach ($resource->properties() as $property) {
+                if ($property == "rdf:type") {
+                    /** @var Resource $item */
+                    foreach ($resource->all("rdf:type") as $item) {
+                        if (preg_match("/omeka\.org\/s\/vocabs\/o/", $item->getUri())) {
+                            $resource->delete("rdf:type", $item);
+                        }
+                    }
+                }
+            }
+            foreach ($resource->propertyUris() as $propertyUris) {
+                if (preg_match("/omeka\.org\/s\/vocabs\/o/", $propertyUris)) {
+                    $resource->delete($propertyUris);
+                }
+            }
+        }
+    }
+
+    protected function dumpSerialisedFiles(Graph $graph): void // candidate for separate class
+    {
+        $fileName = "datacatalog-{$this->id}"; // in separate class make a const FILENAME_PREFIX or so
+        foreach (self::DUMP_FORMATS as $format => $extension) {
+            $content = $graph->serialise($format);
+            file_put_contents(OMEKA_PATH . "/files/datacatalogs/{$fileName}." . $extension, $content);
+            $this->logger->notice(
+                "The file {$fileName}.{$extension} is available." // @translate
```

Comment:
I recon we do not need to translate the log entries? But if the PM specified this differently, consider my feedback nullified.